Settings/AnalyticsOptIn.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {Settings} from 'react-native';
+import TraceTool from './TraceTool';
+
+class AnalyticsOptIn extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      optIn: !!Settings.get('analyticsOptIn'),
```

Comment:
Other settings are saved in asyncStorage, it's better to keep them consistent(there's a helper method in util/asyncstorage). Also Settings only work for iOS.

Reply:
I had originally written this using asyncStorage, modeling it off of `Settings/Notification.js`, but there's no apparent way to read those from iOS native code in `AppDelegate.m`, which is where analytics code is initialized.  Do you think it be useful to store in both asyncStorage and `Settings` / `NSUserDefaults`?  I originally wrote it this way, and I wasn't able to see the benefit, so I took it out, but I can put it back if you'd like.  But the use of `Settings` would remain.

Reply:
I see. Make sense. Then current way is good.